compiler/rustc_codegen_gcc/src/errors.rs

```diff
@@ -0,0 +1,244 @@
+use rustc_errors::{DiagnosticArgValue, IntoDiagnosticArg};
+use rustc_macros::Diagnostic;
+use rustc_middle::ty::Ty;
+use rustc_span::{Span, Symbol};
+use std::borrow::Cow;
+
+struct ExitCode {
```

Comment:
```suggestion
struct ExitCode(Option<i32>);
```
nit: a tuple struct might be a little nicer here